### ISSUE TITLE
Imagenet test 

### ISSUE THREAD

**Issue by jmluu** [hongyi-zhang/Fixup#2]
I have tried to run imagenet with your fixup resnet model, but it did not converge at all. I did not use mix-up method, and all hyper-parameters are defaults.

**Comment by hongyi-zhang** [comment]
Thanks for your feedback. Could you paste the logger output of the first few iterations (until it blows up)?

**Comment by jmluu** [comment]
![log](https://user-images.githubusercontent.com/30696153/54990112-e723ac00-4ff4-11e9-8c7e-5bfdefb3481a.JPG)
From the log we can see that the loss  almost unchanged. By  visualizing data's distribution,  the gradients of most parameters are zero. So that this model can’t learn.
![dist1](https://user-images.githubusercontent.com/30696153/54991755-94e48a00-4ff8-11e9-9b2f-f21f7a500b49.JPG)
![dist2](https://user-images.githubusercontent.com/30696153/54991756-94e48a00-4ff8-11e9-86fb-92805c4c914f.JPG)
![dist3](https://user-images.githubusercontent.com/30696153/54991757-94e48a00-4ff8-11e9-9548-e791967446d5.JPG)

**Comment by hongyi-zhang** [comment]
Thanks. May I know what the x-axis represents? Does it represent number of steps/minibatches? At Step 1 most gradients are supposed to be zero; however they should not be zero after the first step.

**Comment by jmluu** [comment]
Yeah, The x-axis represents the number of steps.

**Comment by hongyi-zhang** [comment]
@Lumonk Was your problem solved?
The repo now contains training scripts to reproduce our ImageNet results -- you may want to take a look :)
Let me know if you have any questions. Thanks!
